packages/react-core/src/components/Dropdown/DropdownMenu.tsx

```diff
@@ -12,8 +12,6 @@ export interface DropdownMenuProps {
   className?: string;
   /** Flag to indicate if menu is opened */
   isOpen?: boolean;
-  /** Flag to indicate if menu should be opened on enter */
-  openedOnEnter?: boolean;
```

Comment:
I checked [patternfly-analytics](https://github.com/rh-uxd/patternfly-analytics) and none of our known consumers are using this prop.

However, since we export this component in `Dropdown/index.ts` it's better to just mark it as deprecated and destructure it so that it's not passed to the DOM. If we remove it entirely we might break some Typescript consumer who is using it, or some Javascript consumer might be confused why their MenuComponent has `openedOnEnter` passed to it which propagates down to an HTMLElement.

Reply:
I see your point. It's sad that we have to keep a property which literally does nothing but you are completely right.

Reply:
@redallen btw this component is not exposed at all in the documentation website. https://www.patternfly.org/v4/documentation/react/components/dropdown

I would say then that since it's not there, it's properties change should not worry us.